modules/drivers/mcp230/mcp230.js

```diff
@@ -233,9 +199,8 @@ class Pin {
     const offset = this.pin >> 3;
     const pin = offset ? this.pin - 8 : this.pin;
     const pinMask = 1 << pin;
-    const IODIR = this.expander.reg.IODIR + offset;
-    const GPIO = this.expander.reg.GPIO + offset;
-
+    const IODIR = this.expander.IODIR + offset;
+    const GPIO = this.expander.GPIO + offset;
 
```

Comment:
Updates per **Device specific constants** section of https://github.com/Moddable-OpenSource/moddable/pull/21#issuecomment-353265714